lib/node-token-validation/src/config/Configuration.ts

```diff
@@ -7,31 +7,70 @@ import { INetworkModule, LoggerOptions, LogLevel, ProtocolMode } from "@azure/ms
 import { Constants } from "../utils/Constants";
 import { NetworkUtils } from "../utils/NetworkUtils";
 
+/**
+ * This object allows you to configure elements of the TokenValidator's functionality, and is passed into the constructor of TokenValidator.
+ */
 export type Configuration = {
+    /**
+     * This is where you configure auth elements like the authority used, the protocol used, and whether there is a clock skew.
+     */
     auth?: TokenValidationOptions,
+    /**
+     * This is where you configure the logger and network client.
+     */
     system?: SystemOptions
 };
 
+/**
+ * Used this to configure the auth options in the Configuration object
+ */
 export type TokenValidationOptions = {
+    /**
+     * Specific authority, used for obtaining keys from the metadata endpoint. Usually takes the form of `https://{uri}/{tenantid}`.
+     */
     authority?: string,
     /**
-     * An array of URIs that are known to be valid. Used in B2C scenarios.
+     * Enum that represents the protocol used by the TokenValidator. 
+     * Used for configuring proper endpoints. 
+     * When set to `OIDC`, the library will not include `/v2.0/` in the authority path when fetching authority metadata. 
+     * When set to `AAD`, the library will include `/v2.0/` in the authority path when fetching authority metadata.
+     */
+    protocolMode?: ProtocolMode,
+    /**
+     * Clock skew (in seconds) allowed in token validation. Must be a positive integer.
```

Comment:
Looks like clock skew and knownAuthorities got swapped

Reply:
Thanks for the catch! Changed.